frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -52,86 +49,94 @@ const getIconClass = (status: ClusterUpdateStatus) => {
-  const currentlyUpdating = isProgressing(cv) || updateFailing(cv);
-  const titleText = currentlyUpdating ? 'Other updates are available.' : 'Cluster update is available.';
-  const buttonText = currentlyUpdating ? 'Update to a different version.' : 'Update now.';
-  return <div className="alert alert-info">
-    <i className="pficon pficon-info" aria-hidden={true} />
-    <strong>
-      {titleText}
-    </strong>
-    <Button bsStyle="link" className="co-m-modal-link" onClick={()=> (clusterUpdateModal({cv}))}>
-      {buttonText}
-    </Button>
-  </div>;
-};
-
-
 const UpdateStatus: React.SFC<UpdateStatusProps> = ({cv}) => {
   const status = getClusterUpdateStatus(cv);
   const iconClass = getIconClass(status);
-  return <React.Fragment>
-    {
-      status === ClusterUpdateStatus.UpdatesAvailable
-        ? <Button bsStyle="link" className="co-m-modal-link" onClick={() => (clusterUpdateModal({cv}))}>
-          <i className={iconClass} aria-hidden={true}></i>
-          &nbsp;
-          {status}
+  if (status === ClusterUpdateStatus.UpdatesAvailable) {
+    return <React.Fragment>
+      <div className="co-update-status">
+        {iconClass && <i className={classNames(iconClass, 'update-pending')} aria-hidden={true}></i>}
+        &nbsp;
+        Update available
+      </div>
+      <div>
+        <Button bsStyle="primary" onClick={() => (clusterUpdateModal({cv}))}>
+          Update now
         </Button>
-        : <span>
-          {iconClass && <i className={iconClass} aria-hidden={true}></i>}
-          &nbsp;
-          {status}
-        </span>
-    }
-  </React.Fragment>;
+      </div>
+    </React.Fragment>;
+  }
+  if (status === ClusterUpdateStatus.Updating) {
+    const updatingCondition = getClusterVersionCondition(cv, ClusterVersionConditionType.Progressing, K8sResourceConditionStatus.True);
+    return <React.Fragment>
+      <div>
+        {iconClass && <i className={iconClass} aria-hidden={true}></i>}
+        &nbsp;
+        Working towards
+      </div>
+      <div>updatingCondition.message && `${updatingCondition.message}. `</div>
+      <Link to="/settings/cluster/clusteroperators?orderBy=desc&sortBy=Status">View detailed progress</Link>
+    </React.Fragment>;
+  }
+  if (status === ClusterUpdateStatus.ErrorRetrieving) {
+    return <React.Fragment>
+      <div>
+        {iconClass && <i className={iconClass} aria-hidden={true}></i>}
+        &nbsp;
+        Error retrieving
+      </div>
+      <Link to="/settings/cluster/clusteroperators?orderBy=desc&sortBy=Status">View detailed progress</Link>
+    </React.Fragment>;
+  }
+  if (status === ClusterUpdateStatus.Failing) {
+    return <React.Fragment>
+      <div>
+        {iconClass && <i className={iconClass} aria-hidden={true}></i>}
+        &nbsp;
+        {status}
+      </div>
+      <Link to="/settings/cluster/clusteroperators?orderBy=desc&sortBy=Status">View detailed progress</Link>
```

Comment:
I'd remove sorting by status. If we do that, the table rows reorder constantly.

```suggestion
      <Link to="/settings/cluster/clusteroperators">View detailed progress</Link>
```